tests/integration-tests/test_runner.py

```diff
@@ -65,6 +65,8 @@
     "post_install": None,
     "vpc_stack": None,
     "cluster": None,
+    "api_definition_s3_uri": "",
```

Comment:
nit: shall we default to None like done for the other params?

Reply:
Agreed